### PR TITLE
U/pingraham/from google doc

### PR DESCRIPTION
**Pull request by patrickingraham**

### CODE REVIEW

File: LSE-509.tex [hunk 1]
```diff
@@ -43,6 +50,659 @@
...
+        \item Rubin Observatory System Specifications (document \citeds{LSE-30})
+
+        \item Rubin Document Tree (document \citeds{LSE-39})
+
+    \end{enumerate}
+
+\textbf{Document Scope and Purpose}
+\label{sec:Scope}
+
+This document describes the overall structure, lines of decision making authority, primary communication channels, task management and workflow for the Rubin Observatory SIT-Com team.
```
Comment (patrickingraham) [inline LSE-509.tex:80]: This is a comment.

**Comment by cclaver** [comment]
Pull request is ready for merge.

**Review by cclaver — APPROVED** [review]
Revisions to the document are ready to be merged.

File: LSE-509.tex [hunk 2]
```diff
@@ -1,4 +1,6 @@
-\documentclass[SE,toc]{lsstdoc}
+%\documentclass[SE,toc]{lsstdoc}
+\documentclass[SE,lsstdraft,authoryear,toc]{lsstdoc}
```
Comment (womullan) [inline LSE-509.tex:2]: ok

**Review by womullan — APPROVED** [review]
please merge this to main so we know which draft to look at

**Pull request merged**